app/views/users/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<section>
+<div class='one_user'>
+    <div class='profile'>
+      <img src='<%= @user.photo %>' alt='<%= @user.name %> thumbnail' />
+    </div>
```

Comment:
- [ ] Kindly make sure you fix the overlapping profile image on each **[wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)** to match the required design. Right now, the profile image on each wireframe makes your design not match the expected result.
You can see on the image of the previous comment.